src/state/authentication/sagas.ts

```diff
@@ -101,9 +120,25 @@ export function* createTxPasswordSaga(action: CreateTxPasswordAction | unknown)
   }
 }
 
+export function* checkTcSaga() {
+  const tcValue = yield call(StoreService.getStoreValue, CONST.tcValue);
+  const tcVersion = yield call(StoreService.getStoreValue, CONST.tcVersion);
+  if (tcValue && Number(tcVersion) >= CONST.tcVersionRequired) {
```

Comment:
` tcValue` seems to be not needed in the whole solution. You can make condition like 
` if (tcVersion && Number(tcVersion) >= CONST.tcVersionRequired) {`

Reply:
can be but again this scenario can change in the next steps will left

Reply:
done with out change left for next step scenario

Reply:
so how in the next scenario this value will be needed?

Reply:
example in settings flow

Reply:
can you explain it in a more detailed way?

Reply:
give up going change and make u version, in next iteration will back to this flow

Reply:
done

Reply:
:D I'm not taking it is a bad solution but I just wanted to know the exact future use case

Reply:
done